hathor/transaction/storage/binary_storage.py

```diff
@@ -153,6 +163,25 @@ def get_tx(hash_bytes, path):
                     yield tx
 
     def get_count_tx_blocks(self):
-        files = os.listdir(self.path)
+        files = os.listdir(self.tx_path)
         assert len(files) % 2 == 0
         return len(files) // 2
+
+    def enable_full_verification(self) -> None:
```

Comment:
I think we shouldn't do anything so specific. Why don't we just create a generic `Registry` that has a given number of parameters? So, we can store other things there in the future.

Reply:
My idea was to create a way to add any key/value to our storages, that's why I created another folder 'attributes' for it. I just left everything in the same method because we are using it only here but maybe I should create another generic method, e.g.

```
def add_key_value(self, key, value):
    filepath = os.path.join(self.attributes_path, key)
    self.save_to_json(filepath, value)

def enable_full_verification(self):
    self.add_key_value(settings.RUNNING_FULL_VERIFICATION_ATTRIBUTE, True)

```

Reply:
I really prefer this way. And I like creating a new subfolder (or just a new file for it) because it allows us to put all the code in `BaseTransactionStorage` without specifics for each storage type.

Reply:
Did this refactor in this commit: https://github.com/HathorNetwork/hathor-core/pull/89/commits/44d1197720ee7eb84554207baa8f57cb398a32dc

Can you see if it's good?

Reply:
It is. Thanks!